gnucash/report/trep-engine.scm

```diff
@@ -263,6 +286,12 @@ in the Options panel."))
               (cons 'text (G_ "Notes"))
               (cons 'renderer-fn (compose xaccTransGetNotes xaccSplitGetParent)))
 
+        (list 'hashtag
+              (cons 'sortkey #f)
+              (cons 'split-sortvalue (lambda (s) (split-hashtag s)))
```

Comment:
a `(lambda (s) (proc s))` is simply reduced to `proc`

Reply:
Good point. I put a lambda because I actually need to pass a second argument to differentiate the sortvalue from the renderer. For sorting reason, I need to put a bunches of ZZZZZ in front of the sortvalue for the unclassified/no tag category. It's a bit dirty but I don't really want to have to rewrite the sort code just to push that one section to the bottom. But the renderer should not show it (it does right now). So in my next iteration I will put an additional #t flag for the sortvalue (lambda (s) (split-hashtag s #t)) vs the renderer (lambda (s) (split-hashtag s #f)) so I can skip the ZZZZ for the renderer.